src/app/_components/_sharedcomponents/Timer/timerUtils.ts

```diff
@@ -0,0 +1,9 @@
+export const formatMilliseconds = (ms: number): string => {
+    const totalSeconds = Math.floor(ms / 1000);
+    const minutes = Math.floor(totalSeconds / 60);
+    const seconds = totalSeconds % 60;
+    return `${minutes.toString()}:${seconds.toString().padStart(2, '0')}`;
+}
+
+export const MAX_TURN_TIME = 60_000; // 60 seconds in ms
```

Comment:
The current turn time limit is 60s in the server code, but I'd heard mention of 30s.